packages/next/src/utilities/createPayloadRequest.ts

```diff
@@ -41,30 +39,14 @@ export const createPayloadRequest = async ({
   }
 
   const urlProperties = new URL(request.url)
-  const { pathname, searchParams } = urlProperties
+  const { pathname } = urlProperties
 
   const isGraphQL =
     !config.graphQL.disable && pathname === `${config.routes.api}${config.routes.graphQL}`
 
-  const { data, file } = await getDataAndFile({
-    collection,
-    config,
-    request,
-  })
-
   let requestFallbackLocale
   let requestLocale
```

Comment:
These variables should be removed from here and below as they are no longer assigned.